_includes/manuals/1.10/4.9_monitoring.md

```diff
@@ -0,0 +1,5 @@
+To monitor your infrastructure, host statuses are useful. In Foreman each host
+has a *global* status that tells which hosts need attention. Each host also has
+sub-statuses that represents status of a particular feature. With every change
+of sub-status a global status is recalculated and the result is determined by
```

Comment:
"With any change of a sub-status, the global status is [..]", or "On any sub-status change, the global status is [..]"